packages/next/index.js

```diff
@@ -0,0 +1,96 @@
+module.exports = {
+    parser: "@babel/eslint-parser",
+    plugins: ["next"],
+    extends:["@sharegate/eslint-config-recommended", "@sharegate/eslint-config-sort-imports"],
```

Comment:
1) Instead of duplicating all the react rules, have you consider extending sharegate eslint react config instead of recommended?

2) Now that [VS Code support sorting the imports](https://code.visualstudio.com/docs/languages/typescript#_organize-imports) I don't think that we still need to use eslint for this.